app/merginprojectmodel.cpp

```diff
@@ -107,8 +109,17 @@ void MerginProjectModel::resetProjects( const MerginProjectList &merginProjects
       // TODO: what else to copy?
     }
 
+    if ( pendingProjects.contains( fullProjectName ) )
+    {
+
+      TransactionStatus projectTransaction = pendingProjects.value( fullProjectName );
+      project->progress = projectTransaction.transferedSize / projectTransaction.totalSize;
+      project->pending = project->progress >= 0;
```

Comment:
shouldn't this be `project->pending = true` if it is listed within pendingProjects ?